src/module_PI/Raspberry_PI/JUnitTests/PacketMessageTranslatorTest.java

```diff
@@ -0,0 +1,124 @@
+package module_PI.Raspberry_PI.JUnitTests;
+
+import static org.junit.Assert.*;
+
+import org.junit.*;
+import org.junit.rules.ExpectedException;
+
+import module_PI.Raspberry_PI.main.PacketMessageTranslator;
+
+public class PacketMessageTranslatorTest {
+	
+	@Rule
+	public ExpectedException expectedEx = ExpectedException.none();
+
+	@Test
+	public void testTranslateFromInt() {
+		int executeCommandInt = 1;
+		PacketMessageTranslator translator = new PacketMessageTranslator();
+		assertEquals(translator.translate(executeCommandInt), "Execute Command: Arm");
+	}
+	
+	@Test
+	public void testTranslateFromInt_TooHigh() {
+		int tooLargeDetailInt = 100000;
+		PacketMessageTranslator translator = new PacketMessageTranslator();
```

Comment:
Thanks for the feedback! What I tried was just declaring one up at the top and having it apply to all the tests, but that didn't work. I'll check out that junit before functionality tonight.